Philips.CodeAnalysis.Test/MsTest/TestContextAnalyzerTest.cs

```diff
@@ -1,12 +1,15 @@
 ﻿// © 2022 Koninklijke Philips N.V. See License.md in the project root for license information.
 
 using System;
+using System.Linq;
 using System.Runtime.InteropServices;
+using System.Security.Cryptography.X509Certificates;
```

Comment:
Using not used

Reply:
Correct. I still need to wire up dotnet-format to the CI/CD pipeline. There's a lot of Code Cleanup stuff still lingering.